src/Microsoft.Azure.SignalR.Management/DependencyInjectionExtensions.cs

```diff
@@ -22,11 +22,19 @@ internal static class DependencyInjectionExtensions
         /// </summary>
         public static IServiceCollection AddSignalRServiceManager(this IServiceCollection services)
         {
-            //cascade options setup
-            services.SetupOptions<ServiceManagerOptions, ServiceManagerOptionsSetup>();
-            services.SetupOptions<ContextOptions, CascadeContextOptionsSetup>();
+            return services.AddSignalRServiceManager<ServiceManagerOptionsSetup>();
+        }
+
+        /// <summary>
+        /// Adds SignalR Service Manager to the specified services collection.
+        /// </summary>
+        /// <param name="services"></param>
```

Comment:
>services [](start = 25, length = 8)

description...

Reply:
This extension won't be public in the future, and the description for this parameter seems trivial.

Reply:
describe all is better, otherwise it looks like someone forget to add it